sql/01.create-user-table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE users (id VARCHAR(64), name VARCHAR(128), password_hash TEXT, email TEXT);
```

Comment:
Thought the maximum length of the email declared in RFC was 256.

Reply:
Indeed. It seems that it must be 254 characters or less.
I'm going to fix it.

https://tools.ietf.org/html/rfc5321#section-4.5.3